### DISCUSSION TITLE
New Costume Editor

### DISCUSSION THREAD

**Discussion by Azulcat** [TurboWarp/extensions#980] [Ideas]
A cosume editor made specifically for desiging levels like the one in gamemaker 2
![6 (1)](https://github.com/TurboWarp/extensions/assets/134739700/4f59467f-e2c9-42fe-8340-cfe9b07f3319)

**Comment by Azulcat** [comment]
https://github.com/TurboWarp/extensions/assets/134739700/344b2c4e-e76e-439b-aa19-91920e95a1a9

**Reply by snowboyz0825** [reply]
I'm sure you can find something similar that let's you export as a png or even maybe a jpeg

**Comment by PURPL3-CAT** [comment]
This looks cool!
